eve/main.yml

```diff
@@ -856,13 +870,14 @@ stages:
       - SetPropertyFromCommand: *set_version_prop
       - SetPropertyFromCommand: *set_short_version_prop
       - TriggerStages:
-          name: Trigger post-merge lifecycle and solutions stages simultaneously
+          name: Trigger post-merge lifecycle, solutions, restore and publish stages simultaneously
           stage_names:
             - post-merge-solutions
             - lifecycle-dev-branch
             - lifecycle-patch-version
             - lifecycle-minor-version
             - bootstrap-restore
+            - publish-images
```

Comment:
We wanted this step to be optional, so that a manual trigger of `post-merge` will not publish anything by default, only when triggered by the nightly schedule.